components/generator/imagevector/src/main/kotlin/io/github/composegears/valkyrie/generator/imagevector/ImageVectorGenerator.kt

```diff
@@ -17,9 +17,7 @@ enum class OutputFormat(val key: String) {
     ;
 
     companion object {
-        fun from(key: String?): OutputFormat {
-            return OutputFormat.valueOf(key ?: BackingProperty.key)
-        }
+        fun from(key: String?) = OutputFormat.entries.find { it.key == key } ?: BackingProperty
```

Comment:
Why need this update?

Reply:
@vkatz mentioned about runtime crash in this place, but I can't see the reason of it. Will check one more time

Reply:
May cause the param isn't `null` but `""`.

Reply:
yes, looks like `var outputFormat: String? by string()` for new install return `""` instead `null`

Reply:
OutputFormat.valueOf takes 'enum name' and not a 'key' param, fun suplies exatly 'key' so it should be searched by 'key'

Reply:
Make sense in this case.